src/components/test-results/test-results.js

```diff
@@ -0,0 +1,135 @@
+import CaretSvg from "../../images/icons/caret-down.svg";
+import "./test-results.styles.scss";
+import { Label } from "../label/label";
+import format from "date-fns/format";
+import fromUnixTime from "date-fns/fromUnixTime";
+
+export const TestResults = ({ personalData = {}, results = [] }) => {
+  const xAxisData = results.map(item =>
+    format(fromUnixTime(item.date), "dd.MM.yyyy / HH:mm")
+  );
+  const yAxisData = results.map(item => ({
+    value: item.temperature,
+    itemStyle: { color: item.temperature > 37 ? "red" : "blue" }
+  }));
+
+  const option = {
+    xAxis: {
+      type: "category",
+      data: xAxisData,
+      axisLine: { show: false, lineStyle: { color: "#fff" } },
+      axisLabel: {
+        textStyle: { color: "#000" }
+      },
+      splitLine: {
+        show: false
+      }
+    },
+    yAxis: {
+      type: "value",
+      axisLine: { show: true, lineStyle: { color: "#B0BEC5" } },
+      axisLabel: { show: false },
+      splitLine: {
+        show: false
+      }
+    },
+    series: [
+      {
+        data: yAxisData,
+        type: "bar",
+        barWidth: "10",
+        markLine: {
+          lineStyle: { width: 0.5, color: "#8CA0B3", style: "dashed" },
+          data: [{ name: "hightemp", yAxis: 37 }],
+          symbol: ["none", "none"]
+        }
+      }
+    ]
+  };
+
+  return (
+    <div className="container">
+      <div className="personal-data">
+        <table className="table personal-data-table">
+          <tbody>
+            <tr>
+              <th>Vârstă</th>
+              <td>{personalData.age}</td>
+            </tr>
+            <tr>
+              <th>Sexul</th>
+              <td>{personalData.gender}</td>
+            </tr>
+            <tr>
+              <th>Fumător/fumătoare</th>
+              <td>{personalData.smoking ? "Da" : "Nu"}</td>
+            </tr>
+            <tr>
+              <th>Afectiuni agravante</th>
+              <td>{personalData.comorbidities.join(", ")}</td>
+            </tr>
+            <tr>
+              <th>În izolare</th>
+              <td>{personalData.inIsolation ? "Da" : "Nu"}</td>
+            </tr>
+            <tr>
+              <th>Alte persoane in casă</th>
+              <td>{personalData.othersInHousehold ? "Da" : "Nu"}</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+
+      <hr style={{ width: "80%", margin: "auto" }} />
+
+      <div className={"section"}>
+        <Label text="Monitorizare temperatura" />
+        <ReactEcharts option={option} height={300} />
+      </div>
+
+      <div className={"section"}>
+        <Label text="Istoric Formular" />
+        <table className="table result-history-table">
+          <thead>
+            <tr>
+              <th>Data/ora</th>
+              <th>Peste 37 grade</th>
+              <th>Durere in gat si/sau dificultate in a inghiti?</th>
+              <th>Tuse intensa</th>
+              <th>Dificultate in a respira</th>
+              <td></td>
```

Comment:
Same here. I think it's best to allow defining the columns from outside the component.

Reply:
@moonflare Then we would need to assume that the data is all strings. Otherwise we can't format dates and such in the component if we loop over the object and display keys and values.

Reply:
unless you mean just passing the labels of the columns along with the value?